avrotest/avrotest_test.go

```diff
@@ -0,0 +1,47 @@
+// +build go1.14
+
+package avrotest
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
```

Comment:
Out of curiosity, why an additional lib for asserting ?

Reply:
It's what Rog used here and I really like it, it's really great.